alacritty/src/cli.rs

```diff
@@ -137,7 +137,8 @@ impl Options {
                 Arg::with_name("class")
                     .long("class")
                     .takes_value(true)
-                    .help(&format!("Defines window class on Linux [default: {}]", DEFAULT_NAME)),
+                    .value_names(&["instance", "general"])
+                    .help(&format!("Defines window class hint on Linux [default: {},{}]", DEFAULT_NAME, DEFAULT_NAME)),
```

Comment:
It looks like this only offers instance **and** general class as suggestions, however it should be clear that it is possible to specify just one of them.

The help output also instructs the user to use it as `--class instance general`, however it should be used as `--class instance,general`.

Reply:
Got it, let me look how to do that and update the PR.  BTW Glad to help

Reply:
@chrisduerr I was able to make the change to show
```
OPTIONS:
        --class <instance>,<general>               Defines window class hint on Linux [default: Alacritty,Alacritty]
```
is this correct or are we looking for below (matching man page)
```
OPTIONS:
        --class [ <instance> | <instance>,<general> ]               Defines window class hint on Linux [default: Alacritty,Alacritty]
```

Reply:
You've fixed 50% of the existing problems. We want to match the manpage.

Reply:
ok one min will try to get the other 50%